help.in

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -u
+
+GROUP="$(id -gn)"
+
+ADMIN="false"
+OPERATOR="false"
+if [[ ${GROUP} == %GROUP_ADMIN% ]]; then
+  ADMIN="true"
+  OPERATOR="true"
+elif [[ ${GROUP} == %GROUP_OPERATOR% ]]; then
+  OPERATOR="true"
+fi
+
+${OPERATOR} && echo -e "power       Host power control"
+${ADMIN}    && echo -e "led         Chassis LED identify control"
+               echo -e "console     Open host console"
+               echo -e "sensors     Print sensors state"
+${ADMIN}    && [[ -e /usr/bin/nmi ]] && echo -e "nmi         Initiate kdump on the host system"
+${ADMIN}    && echo -e "logs        Print BMC system logs"
+${ADMIN}    && echo -e "sosreport   Create system state report"
+${ADMIN}    && echo -e "esel        Decode eSEL"
+${ADMIN}    && echo -e "user        User managment"
+${ADMIN}    && echo -e "update      Update BIOS/BMC firmware"
+${ADMIN}    && echo -e "config      Save/restore/reset BMC configuration"
```

Comment:
I don't like the idea of hard-coding the commands in the `help` script.
There must be a generic method of listing all the installed commands, along with their privileges, and each one of them must be able to provide a short description. To make things faster, their output could be gathered once (maybe even during firmware build by the `install` script) and stored in a file that the `help` script would display.

For example, the generated help file could look like:
```
led "Chassis LED identify control"
...
config "Save/restore/reset BMC configuration"
```
and then the help script could list all the installed scripts along with their groups (i.e. `priv-admin`, `priv-operator`, `priv-user`) and take the descriptions from the the help file filtering it by the current user's privilege.

Reply:
I don't like current implementation too. Let's put it off until the next iteration, I need time to think about it.

Reply:
Just don't forget about it.

Reply:
Fixed. For now, help is constructed dynamically during build.